packages/core/util/__snapshots__/calculateDynamicBlocks.test.ts.snap

```diff
@@ -20,7 +20,6 @@ exports[`five 1`] = `
     "regionNumber": 0,
     "reversed": false,
     "start": 26.05,
-    "variant": undefined,
```

Comment:
something about the modified babel config removed these. probably it is not of consequence...